infra/recipes/azure-sandbox/config/vm-services.tf

```diff
@@ -0,0 +1,62 @@
+// define the virtual network
+resource "azurerm_virtual_network" "sandbox" {
+  name                = "sandbox-network"
+  address_space       = ["10.0.0.0/16"]
```

Comment:
Is this a standard address range or can change?

Reply:
I believe it can change